website/docs/docs/use-dbt-semantic-layer/setup-sl.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Set up the dbt Semantic Layer"
+id: setup-sl
+description: "You can set up the dbt Semantic Layer in dbt Cloud."
+sidebar_label: "Set up the dbt Semantic Layer"
+---
+
+<VersionBlock firstVersion="1.6">
+
+:::info Upgrade your dbt version to access the dbt Semantic Layer
+
+The dbt Semantic Layer has undergone a [significant revamp](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/), making it more efficient to define and query metrics.
+
+**What’s changed?** The dbt_metrics package has been [deprecated](https://docs.getdbt.com/blog/deprecating-dbt-metrics) and replaced with [MetricFlow](/docs/build/about-metricflow?version=1.6), one of the Semantic Layer's key component and a new way framework for defining metrics in dbt.
+
+:::
+
+With the dbt Semantic Layer, you'll be able to centrally define business metrics, reduce code duplication and inconsistency, create self-service in downstream tools, and more. Configure the dbt Semantic Layer in dbt Cloud to connect with your integrated partner tool. 
+
+## Prerequisites
+
+Before you set up the dbt Semantic Layer, make sure you meet the following:
+
+import SetUp from '/snippets/_v2-sl-prerequisites.md';
+
+<SetUp />
+
+
+<Snippet src="sl-considerations-banner" />
+
+
+## Set up dbt Semantic Layer
```

Comment:
need updated images and instructions to new set up

Reply:
@mirnawong1  do you need screenshots for set up here - is that what you mean?

Reply:
i did! but i've got access to the sandbox so i can do it now 🙌